### DISCUSSION TITLE
Locally unzipping/unwinding DNA

### DISCUSSION THREAD

**Discussion by famrein** [BradyAJohnston/MolecularNodes#275] [Q&A] [answered]
Hello, sorry for my insistence with the DNA nodes 😅. My protein is a helicase and im trying to make it so in the center of the protein there is a local unwinding and unzipping. Ive seen your DNA in molecular nodes tutorial and theres an object effect node that does just that. However the current version of molecular nodes does not have that node. From what i can gather, and im not very good at geometry nodes at all, I can use the object info node to extract the position (could be an empty or just the volume of the protein, i guess? theres a volume output from the style_surface_single node on the protein). 

I have the following setup currently, using an empty sphere:

![image](https://github.com/BradyAJohnston/MolecularNodes/assets/22937598/d5266c28-a9bf-49e4-a3d6-6000abe64652)

Moving the sphere closer to the DNA does nothing, so i guess this is wrong. For now i just want the DNA atoms close to the sphere to dissapear. Am I on the right track? Is there a name for this in just regular blender animation that i can look up and learn? My google searches didnt bring much. 

Googling around a bit more I found a mention of the node_append_file.blend file. If I look at this file, there seems to be a node called MOL_anim_falloff_empty that is not included in the actual available nodes. This appears to be what im looking for, though. Is there any way to add it? Or did you not add it because it doesn't work?

Thanks in advance and sorry for the spam!

**Comment by BradyAJohnston** [comment, marked as answer]
Happy to answer questions! 

You have the right idea. You can use the geometry proximity node if you are doing proximity to an actual mesh, but if you are instead using an empty they aren't actual geometry, so you can use a distance calculation instead.

![image](https://github.com/BradyAJohnston/MolecularNodes/assets/36021261/13c3908e-ea2b-4775-a510-5c2b6332fbce)

This setup should get you the result you are after, which will scale the affected distance based on the scale of the empty as well.  Let me know how you go!

**Reply by famrein** [reply]
That worked great, thanks!
